### PR TITLE
W-15809676-org-linking

### PR DESCRIPTION
**Pull request by sarathecoubian**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/external-organization-access.adoc [hunk 1]
```diff
@@ -3,56 +3,11 @@ ifndef::env-site,env-github[]
...
-
-. Click *External Access*.
-. In the *External Access* page, click *Salesforce*. +
-If your organization has at least one established tenant relationship, a list of Salesforce organizations appears, showing the name, organization ID, domain, and status of the relationship connection.
-+
-. Click *Enable* or *Disable*.
+Using the External Access functionality in Access Management, you can:
 
+* xref:trusted-anypoint-platform-org.adoc[Establish a relationship with a trusted Anypoint Platform organization.]
+* xref:trusted-salesforce-org.adoc[Establish a tenant relationship with a trusted Salesforce organization (beta).]
```
Comment (sarathecoubian) [inline modules/ROOT/pages/external-organization-access.adoc]:
```suggestion
* xref:trusted-salesforce-org.adoc[Establish a relationship with a trusted Salesforce organization.]
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+. Click *Connect with a new Salesforce Org*.
+. Enter your connection code from your Salesforce organization.
+. Click *Confirm*. +
+After the connection completes on the Anypoint Platform side, a connection code for your Anypoint Platform organization appears.
++
+. Copy the Anypoint Platform connection code.
+. Return to the Salesforce connection creation interface, and enter your Anypoint Platform connection code.
+. Complete the instructions in Salesforce.
+
+The successful tenant relationship appears in the *Salesforce* section of *External Access* with the `Enabled` status.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
The organization relationship appears in the *Salesforce* section of *External Access* with the `Enabled` status.
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+== Disconnect or Reconnect an Existing Connection
+
+If you want to sever the connection from your Salesforce organization to your Anypoint Platform organization, you can disconnect your Anypoint Platform organization using the Salesforce user interface. If you want to resume publishing assets from Anypoint Platform to Salesforce after you disconnect the Salesforce organization, you must reconnect the organizations. 
+
+Alternatively, if you want to temporarily prevent Anypoint Platform users from publishing assets to Salesforce from the Anypoint Platform side, you can instead disable the connection temporarily.
+
+For more information about managing this relationship from the Salesforce user interface, see the https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Salesforce documentation]. 
+
```
Comment (sarathecoubian) [inline modules/ROOT/pages/trusted-salesforce-org.adoc:54]:
```suggestion

=== Disconnect Your Organizations

. In the Salesforce user interface, select your Anypoint Platform organization.
. Click the overflow menu icon.
. Click *Disconnect*.

=== Reconnect Your Organizations

. In the Salesforce user interface, select your Anypoint Platform organization.
. Click the overflow menu icon.
. Click *Reconnect*.

```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+When you enable the connection with your Salesforce organization, you can publish assets from your Anypoint Platform organization to the trusted Salesforce organization. When you disable the connection, the Salesforce organization still appears in the list of available organizations but you can no longer publish assets from your Anypoint Platform to your Salesforce organization. Disabling a connection does not sever the connection to your Salesforce organization. 
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. Click *External Access*.
+. In the *External Access* page, click *Salesforce*. +
+If your organization has at least one established tenant relationship, a list of Salesforce organizations appears, showing the name, organization ID, domain, and status of the relationship connection.
++
+. Click *Enable* or *Disable*.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/trusted-salesforce-org.adoc:77]:
```suggestion
. Click *Enable* or *Disable*.

== Remove a Connection 

If you want to permanently delete the connection between your Anypoint Platform organization and your Salesforce organization using the Salesforce interface. After you delete the connection, a user with the appropriate permissions in both your Anypoint Platform and Salesforce organizations must recreate the connection.

. In the Salesforce user interface, select your Anypoint Platform organization.
. Click the overflow menu icon.
. Click *Remove*.

```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+== Remove a Connection 
+
+If you want to permanently delete the connection between your Anypoint Platform organization and your Salesforce organization using the Salesforce interface. After you delete the connection, a user with the appropriate permissions in both your Anypoint Platform and Salesforce organizations must recreate the connection.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
You can permanently delete the connection between your Anypoint Platform organization and your Salesforce organization using the Salesforce interface. After you delete the connection, a user with the appropriate permissions in both your Anypoint Platform and Salesforce organizations must recreate the connection if they want to reestablish a relationship.
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 6]
```diff
@@ -0,0 +1,82 @@
+= Establish a Tenant Relationship to a Trusted Salesforce Organizations 
+
+Salesforce organizations can establish a tenant relationship with Anypoint Platform organizations. After establishing this relationship, an Anypoint Platform user with the Organization Administrator permission can enable or disable this relationship in *Access Management*. You can also disconnect Anypoint Platform from Salesforce at any time. For more information about initiating and maintaining a relationship with Salesforce, see https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Manage Your MuleSoft Anypoint Platform Connection] in the Salesforce documentation.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
A Salesforce organization can establish a tenant relationship with an Anypoint Platform organization. After establishing this relationship, an Anypoint Platform user with the Organization Administrator permission can enable or disable this relationship in *Access Management*. You can also disconnect Anypoint Platform from Salesforce at any time. For more information about initiating and maintaining a relationship with Salesforce, see https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Manage Your MuleSoft Anypoint Platform Connection] in the Salesforce documentation.
```

File: modules/ROOT/pages/trusted-anypoint-platform-org.adoc [hunk 7]
```diff
@@ -0,0 +1,34 @@
+= Enable External Access to Trusted Anypoint Platform Organizations
```
Comment (sarathecoubian) [inline modules/ROOT/pages/trusted-anypoint-platform-org.adoc]:
```suggestion
= Enable External Access to a Trusted Anypoint Platform Organization
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 8]
```diff
@@ -0,0 +1,82 @@
+= Establish a Tenant Relationship to a Trusted Salesforce Organizations
```
Comment (sarathecoubian) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
= Establish a Tenant Relationship to a Trusted Salesforce Organization
```

File: modules/ROOT/pages/trusted-anypoint-platform-org.adoc [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+. In the *External Access* page, click *Add organization*.
+. Enter the unique domain name of the organization you want to share permissions with.
+. Click *Add*.
+
+
+== Remove a Trusted Organization
+
+When you remove a trusted organization from your external access configuration, both organizations lose all existing shared permissions. In addition, you cannot grant new permissions until you reestablish the trust relationship.
+
+To remove a trusted organization from your list of trusted organizations:
```
Comment (valkyrie69) [inline modules/ROOT/pages/trusted-anypoint-platform-org.adoc]:
```suggestion
To remove a trusted organization:
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 10]
```diff
@@ -0,0 +1,82 @@
+= Establish a Tenant Relationship to a Trusted Salesforce Organization
+
+A Salesforce organization can establish a tenant relationship with an Anypoint Platform organization. After establishing this relationship, an Anypoint Platform user with the Organization Administrator permission can enable or disable this relationship in *Access Management*. You can also disconnect Anypoint Platform from Salesforce at any time. For more information about initiating and maintaining a relationship with Salesforce, see https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Manage Your MuleSoft Anypoint Platform Connection] in the Salesforce documentation.
+
+Using the Salesforce user interface, you can:
```
Comment (valkyrie69) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
Use the Salesforce user interface to:
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 11]
```diff
@@ -0,0 +1,82 @@
+= Establish a Tenant Relationship to a Trusted Salesforce Organization
+
+A Salesforce organization can establish a tenant relationship with an Anypoint Platform organization. After establishing this relationship, an Anypoint Platform user with the Organization Administrator permission can enable or disable this relationship in *Access Management*. You can also disconnect Anypoint Platform from Salesforce at any time. For more information about initiating and maintaining a relationship with Salesforce, see https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Manage Your MuleSoft Anypoint Platform Connection] in the Salesforce documentation.
```
Comment (valkyrie69) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
When a Salesforce organization establishes a tenant relationship with an Anypoint Platform organization, an Anypoint Platform user with the Organization Administrator permission can then enable or disable this relationship in *Access Management*. 

You can also disconnect Anypoint Platform from Salesforce at any time. 

For more information about initiating and maintaining a relationship with Salesforce, see https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Manage Your MuleSoft Anypoint Platform Connection] in the Salesforce documentation.
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+
+Using the Salesforce user interface, you can:
+
+* Initiate and complete the connection process to your Anypoint Platform organization.
+* Disconnect or reconnect your Anypoint Platform organization from your Salesforce organization.
+* Delete the connection between your Anypoint Platform organization and your Salesforce organization.
+
+Using the Anypoint Platform user interface, you can:
```
Comment (valkyrie69) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
Use the Anypoint Platform user interface to:
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
+* Continue creating a connection to a Salesforce organization.
+* Enable or disable a tenant relationship with Salesforce.
+
+== Before You Begin
+
+You need the following:
```
Comment (valkyrie69) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
You must have the following:
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
++
+. Copy the Anypoint Platform connection code.
+. Return to the Salesforce connection creation interface, and enter your Anypoint Platform connection code.
+. Complete the instructions in Salesforce.
+
+The organization relationship appears in the *Salesforce* section of *External Access* with the `Enabled` status.
+
+== Disconnect or Reconnect an Existing Connection
+
+If you want to sever the connection from your Salesforce organization to your Anypoint Platform organization, you can disconnect your Anypoint Platform organization using the Salesforce user interface. If you want to resume publishing assets from Anypoint Platform to Salesforce after you disconnect the Salesforce organization, you must reconnect the organizations.
```
Comment (valkyrie69) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
If you want to disconnect your Salesforce organization from your Anypoint Platform organization, you can use the Salesforce user interface to do so. If you want to resume publishing assets from Anypoint Platform to Salesforce after you disconnect the Salesforce organization, you must reconnect the organizations. 
```

File: modules/ROOT/pages/trusted-salesforce-org.adoc [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+
+. Click *External Access*.
+. In the *External Access* page, click *Salesforce*. +
+If your organization has at least one established tenant relationship, a list of Salesforce organizations appears, showing the name, organization ID, domain, and status of the relationship connection.
++
+. Click *Enable* or *Disable*.
+
+== Remove a Connection 
+
+You can permanently delete the connection between your Anypoint Platform organization and your Salesforce organization using the Salesforce interface. After you delete the connection, a user with the appropriate permissions in both your Anypoint Platform and Salesforce organizations must recreate the connection if they want to reestablish a relationship.
```
Comment (valkyrie69) [inline modules/ROOT/pages/trusted-salesforce-org.adoc]:
```suggestion
You can permanently delete the connection between your Anypoint Platform organization and your Salesforce organization using the Salesforce interface. After you delete the connection, a user with the appropriate permissions in both your Anypoint Platform and Salesforce organizations must recreate the connection if they want to reestablish a relationship.
```
```suggestion
You can permanently delete the connection between your Anypoint Platform organization and your Salesforce organization using the Salesforce user interface. After you delete the connection, a user with the appropriate permissions in both your Anypoint Platform and Salesforce organizations must recreate the connection if they want to reestablish a relationship.
```

**Review by valkyrie69 — DISMISSED** [review]

**Review by valkyrie69 — DISMISSED** [review]

**Review by valkyrie69 — APPROVED** [review]

File: modules/ROOT/pages/trusted-anypoint-platform-org.adoc [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+
+To add an organization to your configuration, you must know its Anypoint xref:access-management::organization.adoc#manage-root-organization-settings[domain]. Domains are human-readable names that uniquely identify an Anypoint Platform organization. Ask the administrators of the other organization what their domain is, and be prepared to share your domain with them so they can add your organization to their list in return.
+You can view your organization's unique domain name in the *External Access* page in Access Management.
+
+
+To add an external organization to your list of trusted organizations:
+
+. From the *Access Management* home page, click the *External Access* tab.
+. In the *External Access* page, click *Add organization*.
+. Enter the unique domain name of the organization you want to share permissions with.
```
Comment (emileemooreVT) [inline modules/ROOT/pages/trusted-anypoint-platform-org.adoc:19]: Nitpick - Should this be "...the organization with which you want to share...."

**Pull request merged**